### DISCUSSION TITLE
Is it possible to include libsvtav1 in the binary build?

### DISCUSSION THREAD

**Discussion by caffeinism** [PyAV-Org/PyAV#1653] [1. Help] [answered]
I want to do efficient encoding using the av1 codec, but the reference encoder libaom-av1 seems to be very slow in ffmpeg. After some tests and searching, I found that libsvtav1 is included in most ffmpeg binaries and is the codec used for production, and it seems that there is a big difference in speed. Could you please include the codec in the build? Or, if not, is there a way for me to customize it and include it?

**Comment by WyattBlue** [comment]
This seems reasonable to me

**Reply by caffeinism** [reply]
Thank you for your response.

**Reply by caffeinism** [reply]
@WyattBlue  Would it be possible for you to let me know when this might be officially released?

**Reply by JoeSchiff** [reply]
@caffeinism PyAV 14.1.0 has been released and includes libsvtav1.

**Reply by caffeinism** [reply]
thanks! i'll check when I can

**Comment by JoeSchiff** [comment, marked as answer]
https://github.com/PyAV-Org/pyav-ffmpeg/pull/139

libsvtav1 is now bundled with pyav. However, I believe you will need to wait until the next (major?) release for it to be included in `pip install av`

When that happens, please try it out and let me know.

**Reply by caffeinism** [reply]
Thanks, I'll try it out when it's released.

**Reply by caffeinism** [reply]
This works great. I was just wondering though, darwin arm includes v2.3.0 and linux x86 includes v2.2.0, is this by intent?

**Reply by JoeSchiff** [reply]
All platforms should be v2.2.1. How are you checking the version of libsvtav1?

**Reply by caffeinism** [reply]
Oh, I double-checked, I must have confused something with the local version. Thanks!
